### PR TITLE
Create vignette for BioCro regional runs

### PR DESCRIPTION
**Pull request by KristinaRiemer**

### CODE REVIEW

File: BioCro/DARPA/biocro_regional_darpa.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+library(dplyr)
+library(ncdf.tools)
+library(ggplot2)
+library(maps)
+library(gganimate)
+```
+
+## Read in input files
+
+Past run config
```
Comment (Chris-Schnaufer) [inline BioCro/DARPA/biocro_regional_darpa.md]: How does one get the past .XML file?

File: BioCro/DARPA/biocro_regional_darpa.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+How to Do Regional Runs on DARPA Data Using BioCro Model
+================
+Kristina Riemer, University of Arizona
```
Comment (Chris-Schnaufer) [inline BioCro/DARPA/biocro_regional_darpa.md]: Your name under the title looks kind of funny (as in strange). You might want to prefix it with `**Author**:` or something like that

File: BioCro/DARPA/biocro_regional_darpa.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+## Convert weather data
+
+The weather data is a subset of NARR around Champaign, Illinois. It’s
+currently in the Pecan standard format, which is a netCDF file. The
+dimensions are 3 latitude, 6 longitude, and 93,504 time steps, which are
+hourly from 1979 through 2010. There are various weather variables,
+including temperature, precipitation flux, and solar radiation.
+
+This is converted to the met format for BioCro. Data for each of the
+locations (unique latitude and longitude combos) is turned into a flat
```
Comment (Chris-Schnaufer) [inline BioCro/DARPA/biocro_regional_darpa.md]: Perhaps: `combos` -> `combinations`

File: BioCro/DARPA/biocro_regional_darpa.md [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+csv file, with one row per hour for the ~32 years.
```
Comment (Chris-Schnaufer) [inline BioCro/DARPA/biocro_regional_darpa.md]: Perhaps: `~32 years` -> `the specified time range (approximately 32 years)`

File: BioCro/DARPA/biocro_regional_darpa.md [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+
+## Convert weather data
+
+The weather data is a subset of NARR around Champaign, Illinois. It’s
+currently in the Pecan standard format, which is a netCDF file. The
+dimensions are 3 latitude, 6 longitude, and 93,504 time steps, which are
+hourly from 1979 through 2010. There are various weather variables,
+including temperature, precipitation flux, and solar radiation.
+
+This is converted to the met format for BioCro. Data for each of the
```
Comment (Chris-Schnaufer) [inline BioCro/DARPA/biocro_regional_darpa.md]: Is `met` an acronym? Is it capitalized in written text? (No need to answer, just posing the question)

Perhaps: `This is converted ...` -> `The netCDF file format needs to be converted ...`

File: BioCro/DARPA/biocro_regional_darpa.md [hunk 6]
```diff
@@ -0,0 +1,132 @@
...
+  biocro_met_path <- paste0("biocro_regional_files/biocro_met_by_location/biocromet-", 
+                            latlon$lat[point], "-", latlon$lon[point], ".2004.csv")
+  write.csv(biocro_met, biocro_met_path, row.names = FALSE)
+  
+}
+```
+
+## Run BioCro on each location
+
+BioCro is run on each of the locations weather and soil data to produce
```
Comment (Chris-Schnaufer) [inline BioCro/DARPA/biocro_regional_darpa.md]: Perhaps: `BioCro is run on ...` -> `BioCro is now run on ...`

File: BioCro/DARPA/biocro_regional_darpa.md [hunk 7]
```diff
@@ -0,0 +1,132 @@
...
+  biocro_results_daily$lat <- latlon$lat[point]
+  biocro_results_daily$lon <- latlon$lon[point]
+  biocro_results <- rbind(biocro_results, biocro_results_daily)
+  write.csv(biocro_results, "biocro_regional_files/all_biocro_results.csv")
+}
+```
+
+## Plot estimated biomass
+
+The code below produces a video of a map of total biomass (stem, leaf,
```
Comment (Chris-Schnaufer) [inline BioCro/DARPA/biocro_regional_darpa.md]: Perhaps: change `a map` -> `the mapping `

File: BioCro/DARPA/biocro_regional_darpa.md (hunk 5 above)
Comment (KristinaRiemer) [reply BioCro/DARPA/biocro_regional_darpa.md]: Met is short for meteorology.

**Comment by KristinaRiemer** [comment]
Thanks for all the comments @Chris-Schnaufer! As they were all good, I implemented all of them. I did some more revising too, as I had initially just thrown this vignette together in a very bare bones way, so it should be improved now.

**Pull request merged**